docs/tutorials/05_uploading_program.ipynb

```diff
@@ -90,8 +90,8 @@
    "source": [
     "Each runtime program must have a `main()` function, which serves as the entry point to the program. This function must have `backend` and `user_messenger` as the first two positional arguments:\n",
     "\n",
-    "- `backend` is an instance of [ProgramBackend](https://qiskit.org/documentation/partners/qiskit_ibm_runtime/stubs/qiskit_ibm_runtime.ProgramBackend.html#qiskit_ibm_runtime.ProgramBackend) and has a [run()](https://qiskit.org/documentation/partners/qiskit_ibm_runtime/stubs/qiskit_ibm_runtime.ProgramBackend.html#qiskit_ibm_runtime.ProgramBackend.run) method that can be used to submit circuits.\n",
-    "- `user_messenger` is an instance of [UserMessenger](https://qiskit.org/documentation/partners/qiskit_ibm_runtime/stubs/qiskit_ibm_runtime.UserMessenger.html#qiskit_ibm_runtime.UserMessenger) and has a [publish()](https://qiskit.org/documentation/partners/qiskit_ibm_runtime/stubs/qiskit_ibm_runtime.UserMessenger.html#qiskit_ibm_runtime.UserMessenger.publish) method that can be used to send interim and final results to the program user. This method takes a parameter `final` that indicates whether it's a final result. However, it is recommended to return the final result directly from the `main()` function. Currently only final results are stored after a program execution finishes."
+    "- `backend` is an instance of [ProgramBackend](https://qiskit.org/documentation/stubs/qiskit.providers.ibmq.runtime.ProgramBackend.html) and has a [run()](https://qiskit.org/documentation/stubs/qiskit.providers.ibmq.runtime.ProgramBackend.run.html#qiskit.providers.ibmq.runtime.ProgramBackend.run) method that can be used to submit circuits.\n",
+    "- `user_messenger` is an instance of [UserMessenger](https://qiskit.org/documentation/stubs/qiskit.providers.ibmq.runtime.UserMessenger.html) and has a [publish()](https://qiskit.org/documentation/stubs/qiskit.providers.ibmq.runtime.UserMessenger.publish.html#qiskit.providers.ibmq.runtime.UserMessenger.publish) method that can be used to send interim and final results to the program user. This method takes a parameter `final` that indicates whether it's a final result. However, it is recommended to return the final result directly from the `main()` function. Currently only final results are stored after a program execution finishes."
```

Comment:
We should not be pointing to classes in qiskit-ibmq-provider. @y4izus Could you please check why these links are not working after the docs refactor?

Reply:
I've checked why the page for the `ProgramBackend` doesn't exist and it's because it's not listed on the [autosummary](https://github.com/Qiskit/qiskit-ibm-runtime/blob/main/qiskit_ibm_runtime/__init__.py#L216). All classes we want to have on the page, should be listed on the `__init__.py` file, because it's the way Sphinx has to know what pages it should create. 

It looks like none of the classes on the `program` folder are listed on the autosummary section, that's why we don't have pages for `ProgramBackend`, `ResultDecoder` and `UserMessenger`

Reply:
Thanks, if this wasn't removed intentionally then we should add it back https://github.com/Qiskit/qiskit-ibm-runtime/pull/263/files#diff-c2aa5d3070ecd7d355ba8065fc883b6b248f8939be806e4a9d1288cc582fc44c

Reply:
Thanks, if this wasn't removed intentionally then we should add it back https://github.com/Qiskit/qiskit-ibm-runtime/pull/263/files#diff-c2aa5d3070ecd7d355ba8065fc883b6b248f8939be806e4a9d1288cc582fc44c

Reply:
It was decided in a meeting to removed the old API Reference page to have a new one with more info, so that page was removed intentionally. We need the collaboration of content creators to know where they want to see the classes inside the `program` folder. Is it ok to have them on the [classes section](https://qiskit.org/documentation/partners/qiskit_ibm_runtime/apidocs/ibm-runtime.html#classes) or do they prefer another place?

Reply:
@y4izus Who are content creators? Would you be able to co-ordinate with them?